perception/sub8_pointcloud/include/OGridRegionType.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
Is this used at all?

I think it's best to not have multiple copies of this and just use [the one in C3](https://github.com/uf-mil/SubjuGator/pull/242/files#diff-732da72f004e537bc7a7d42305096915R7)